react_juce/core/View.cpp

```diff
@@ -248,18 +248,52 @@ namespace reactjuce
 
     void View::mouseEnter (const juce::MouseEvent& e)
     {
-        dispatchViewEvent("onMouseEnter", detail::makeViewEventObject(e, *this));
+        if (!wasMouseOver)
+        {
+            dispatchViewEvent("onMouseEnter", detail::makeViewEventObject(e, *this));
+        }
+
+        wasMouseOver = true;
     }
 
     void View::mouseExit (const juce::MouseEvent& e)
     {
-        dispatchViewEvent("onMouseLeave", detail::makeViewEventObject(e, *this));
+        if (!reallyContains(e.getPosition(), true))
```

Comment:
reallyContains returns true even if the mouse is over one of this components children.